draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -857,8 +886,62 @@ are not blinded, but contain disclosable claims. This limitation
 needs to be taken into account by Issuers when creating the structure of
 the SD-JWT.
 
+## Issuer Signature Key Distribution and Rotation {#issuer_signature_key_distribution}
+
+This specification does not define how signature verification keys of
+Issuers are distributed to Verifiers. However, it is RECOMMENDED that
+Issuers publish their keys in a way that allows for efficient and secure
+key rotation and revocation, for example, by publishing keys at a
+predefined location using the JSON Web Key Set (JWKS) format [@RFC7517].
+Verifiers need to ensure that they are not using expired or revoked keys
+for signature verification as reasonable and appropriate for the given
+key distribution method.
```

Comment:
```suggestion
Verifiers need to ensure that they are not using expired or revoked keys
for signature verification using reasonable and appropriate method for the given
key distribution method.
```

Reply:
I think the original sentence is just fine.

Reply:
The original sentence seemed okay to me too. And the suggested change isn't quite grammatically correct.

Reply:
I just don't understand the meaning of the original sentence. can you please paraphrase in a grammatically correct way so that I don't have to spend 5min thinking about the meaning..?

Reply:
```suggestion
Verifiers need to ensure that they are not using expired or revoked keys
for signature verification using reasonable and appropriate means for the given
key-distribution method.
```

is that wording more straightforward?